pyrobosim/examples/demo_prm.py

```diff
@@ -29,7 +29,6 @@ def test_prm():
     test_prm()
 
     import sys
-    from pyrobosim.gui.main import PyRoboSimGUI
+    from pyrobosim.gui.main import start_gui
 
-    app = PyRoboSimGUI(w, sys.argv)
-    sys.exit(app.exec_())
+    start_gui(w, sys.argv)
```

Comment:
Should the world variable name be changed from `w` to `world` for consistency?
since we changed it in the astar demo.